app/controllers/admin/site_settings_controller.rb

```diff
@@ -8,8 +9,22 @@ def update
       params.require(:settings).map do |param|
         data = param.to_unsafe_h
         CustomSetting[data['var']] = data['value']
+        setting = CustomSetting.find_by(var: data['var']) || CustomSetting.new(var: data['var'])
+        setting.value = data['value']
+        setting.save
       end
       redirect_to '/admin/site_settings/'
     end
+
+    private
+
+      def save_current
+        keys = %w(site.display_login_link site.only_admin_can_upload site.signup_enabled site.name)
+        keys.each do |k|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'k' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->